includes/hooks.php

```diff
@@ -73,17 +73,26 @@ function vpfo_save_alert_meta( $post_id ) {
 		return;
 	}
 
-	// sanitize post id for db insertion
-	$post_id_sanitized = absint( $post_id );
-
 	// Save the 'display alert' checkbox value
-	$display_alert           = isset( $_POST['vpfo_display_alert'] ) ? '1' : '0';
-	$display_alert_sanitized = esc_html( $display_alert );
-	update_post_meta( $post_id_sanitized, '_vpfo_display_alert', $display_alert_sanitized );
+	$display_alert = isset( $_POST['vpfo_display_alert'] ) ? '1' : '0';
+	update_post_meta( $post_id, '_vpfo_display_alert', $display_alert );
```

Comment:
I'm going to approve this for now. But please, in your next commit, can you escape `$display_alert`? I know it's set to either string 1 or 0 above _right now_ but let's make sure that if the logic for that variable ever changes, we don't have to worry about it being escaped properly when it's hitting the database.